packages/reason-parser/reason-parser.2.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Jordan Walke <jordojw@gmail.com>"
+authors: "Jordan Walke <jordojw@gmail.com>"
+homepage: "https://github.com/facebook/reason"
+bug-reports: "https://github.com/facebook/reason/issues"
+license: "BSD"
```

Comment:
We would like this release to be merged asap so that we can unbreak people and we will go update our packages after discussing with the right people who know about the licensing.